src/content/docs/en/editor-setup.mdx

```diff
@@ -55,7 +55,9 @@ See [the project's User Guide](https://ota-meshi.github.io/eslint-plugin-astro/u
 
 ### Prettier
 
-[Prettier](https://prettier.io/) is a popular formatter for JavaScript, HTML, CSS and more. To add support for formatting `.astro` files, use [the official Astro Prettier plugin](https://github.com/withastro/prettier-plugin-astro).
+If you're using the [Astro VS Code Extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) or [the Astro LSP within another editor](#other-code-editors), code formatting is included. Otherwise, you can use [Prettier](https://prettier.io/), a popular formatter for JavaScript, HTML, CSS and more. 
+
+To add support for formatting `.astro` files, use [the official Astro Prettier plugin](https://github.com/withastro/prettier-plugin-astro).
```

Comment:
Thanks for investigating this, Dan!

This phrasing makes it seem like the built-in formatter is _not_ Prettier:

> If X, code formatting is included. Otherwise, Prettier.

What do you think of this phrasing? I also avoided the “LSP” acronym for clarity:

```suggestion
[Prettier](https://prettier.io/) is a popular formatter for JavaScript, HTML, CSS and more. If you're using the [Astro VS Code Extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) or [the Astro language server within another editor](#other-code-editors), code formatting with Prettier is included.

If you want to use Prettier without Astro’s language server, for example to run [Prettier’s CLI](https://prettier.io/docs/en/cli.html), you can add support for formatting `.astro` files with [the official Astro Prettier plugin](https://github.com/withastro/prettier-plugin-astro).
```

Reply:
LGTM!